### ISSUE TITLE
Video summary needs updating

### ISSUE THREAD

**Issue by Themanwithoutaplan** [HTTPArchive/legacy.httparchive.org#153]
The [video summary](https://httparchive.org/reports/page-weight#bytesVideo) lists only four mime-types for video: mp4, swf, f4v, or flv. However, [research](https://discuss.httparchive.org/t/the-elephant-in-the-room-video/1087) suggests that this is an outdated classification and that WebM and TS should also be included. MP4 now exists in different flavours (AVC/h264 and HEVC/h265) as does WebM (VP8 and VP9), and AV1 is supposed to be the next format for all with support already in Chrome.

Is it possible to add the newer formats to the categories?
Is it possible to recalculate historical values taking these changes into consideration?

**Comment by rviscomi** [comment]
Great questions.

> Is it possible to add the newer formats to the categories?

Yes, should be a straightforward fix: https://github.com/HTTPArchive/legacy.httparchive.org/blob/cca0e5a444b5742a56fc590ad986d24f133d774d/bulktest/batch_lib.inc#L967. Could you submit a PR?

> Is it possible to recalculate historical values taking these changes into consideration?

Possible but quite a bit more involved as it requires querying all requests and replacing all of the summary_pages tables. Not something I have capacity for now.

**Comment by Themanwithoutaplan** [comment]
Yeah, I should be able to submit a PR.
FWIW I've created a PR based on Doug's to identify the extensions and mimetypes that are classified as video. This skips anything using an embedded JS player but the URL list now contains far fewer YouTube webistes.

```
SELECT
lower(ext) as ext, mimeType, count(*) as cnt
  FROM
httparchive.runs.latest_requests
  WHERE
   type CONTAINS "video"
group by mimeType, ext
order by cnt desc
```

**Comment by rviscomi** [comment]
Fixed by #154
